### PR TITLE
custom job id and job number

### PR DESCRIPTION
**Pull request by shogo82148**
related to #122, and #123 

before adding the job number https://coveralls.io/builds/27070546
before

after adding the job number https://coveralls.io/builds/27069677
image

Here is an example of configure for GitHub Actions.

https://github.com/shogo82148/go-sql-proxy/pull/35

https://github.com/shogo82148/go-sql-proxy/blob/bf53365379fe59e5cb5ddde6c54a04b79a81ae74/.github/workflows/go.yml#L32-L46
```yaml
    - name: Send coverage
      env:
        COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      run: |
        GO111MODULE=off go get github.com/shogo82148/goveralls
        $(go env GOPATH)/bin/goveralls -coverprofile=profile.cov -parallel -jobid=$GITHUB_SHA -service=github

  finish:
    needs: test
    runs-on: ubuntu-latest
    steps:
      - name: finish
        env:
          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        run: curl -sS "https://coveralls.io/webhook" -d "repo_token=$COVERALLS_TOKEN&repo_name=$GITHUB_REPOSITORY&payload[build_num]=$GITHUB_SHA&payload[status]=done"
```

### CODE REVIEW

**Comment by mattn** [comment]
So, can I test goverall it-self with this change!?

File: goveralls.go [hunk 1]
```diff
@@ -248,8 +251,14 @@ func process() error {
 	//
 	// Initialize Job
 	//
+
+	// flags are never nil, so no nil check needed
 	var jobId string
-	if travisJobId := os.Getenv("TRAVIS_JOB_ID"); travisJobId != "" {
+	if *customJobId != "" {
+		jobId = *customJobId
+	} else if serviceJobId := os.Getenv("COVERALLS_SERVICE_JOB_ID"); serviceJobId != "" {
```
Comment (shogo82148) [inline goveralls.go]: renamed to `COVERALLS_SERVICE_JOB_ID`
it is compatible with node-coveralls https://github.com/nickmerwin/node-coveralls#usage

File: goveralls.go [hunk 2]
```diff
@@ -82,6 +84,7 @@ type SourceFile struct {
 type Job struct {
 	RepoToken          *string       `json:"repo_token,omitempty"`
 	ServiceJobId       string        `json:"service_job_id"`
+	ServiceJobNumber   string        `json:"service_job_number,omitempty"`
```
Comment (shogo82148) [inline goveralls.go]: Its name is `service_number` on the document https://docs.coveralls.io/api-reference, but doesn't work.
It seems to be a typo.

https://github.com/nickmerwin/node-coveralls/blob/2ed185ca3626591549ed33c2363a47b824a39d0f/lib/getOptions.js#L80

**Comment by shogo82148** [comment]
> So, can I test goverall it-self with this change!?

That's right!

**Comment by mattn** [comment]
YAY!

**Pull request merged**